### ISSUE TITLE
Unable to install CLI on raspberry pi

### ISSUE THREAD

**Issue by japborst** [AssemblyAI/assemblyai-cli#110]
**Describe the bug**
When attempting to install the CLI on a raspberry pi, it fails without any errors

**To Reproduce**
1. SSH into the raspberry pi
2. Execute `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/AssemblyAI/assemblyai-cli/main/install.sh)"`

**Expected behavior**
Installs the CLI

**Experienced behavior**
Prints
```
Using the OSS distribution...
Downloading AssemblyAI CLI v1.17.4...
```

No errors or other information is printed.

**Desktop (please complete the following information):**
```
$ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
```

**Additional context**
n.a.

**Comment by japborst** [comment]
Ah 💡 Looking at your releases, the armv7 releases are missing although the install script is looking for it.

```sh
$ uname -m
armv7l
```

**Comment by fcastillo90** [comment]
Hey @japborst! Currently, we offer support for armv6, although I just created a PR for supporting armv7 and will let you know as soon as we can test and release it.

**Comment by japborst** [comment]
Running `armv6` does also work. Just mapping that in the install script unfortunately results in other errors, but have been able to manually install it as a workaround.

**Comment by fcastillo90** [comment]
Hey @japborst! Yes, the issue was that on the install script we did not have support for `armv7l` architecture. This issue is now addressed on release `v1.17.5`
